lib/elastomer/client/index.rb

```diff
@@ -533,6 +533,17 @@ def delete_by_query(query, params = nil)
         docs.delete_by_query(query, params)
       end
 
+      # Constructs a Percolator with the given id on this index.
+      #
+      # Examples
+      #
+      #   index.percolator "1"
+      #
+      # Returns a Percolator
+      def percolator(id)
+        Percolator.new(@client, @name, id)
```

Comment:
There's a slight inconsistency here compared to the warmer interface. A `Warmer` object can be constructed two ways:

```ruby
client.warmer('index_name', 'warmer_name')
client.index('index_name').warmer('warmer_name')
```

With the above percolator interface, there's no `client.percolator` option.

Warmers and percolators both require an index scope, so I'm open to removing `Client#warmer` if there's agreement that it's not useful. Either way both objects should have similar interfaces.